web/themes/custom/accessibility_theme/src/patterns/components/heading/heading.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../global/utils/init';
+
+.heading__link {
+  color: inherit;
+  text-decoration: none;
+
+  &:hover {
```

Comment:
We will probably do this as a global style.  We can remove.

Reply:
I'm assuming you're referencing the hover effect for this comment so I'll remove that. If you mean the thing in it's entirety, let me know and I'll remove the rest of it in lieu of global styles.

Reply:
Yeah hover styles.